addons/mail/static/src/js/views/activity/activity_controller.js

```diff
@@ -38,17 +39,16 @@ var ActivityController = BasicController.extend({
      */
     _onScheduleActivity: function () {
         var self = this;
-
         var state = this.model.get(this.handle);
         new ViewDialogs.SelectCreateDialog(this, {
             res_model: state.model,
             domain: this.model.originalDomain,
-            title: _.str.sprintf(_t("Search: %s"), this.renderer.arch.attrs.string),
+            title: _.str.sprintf(_t("Search: %s"), this.renderer.props.arch.attrs.string),
```

Comment:
if the controller needs this information (the arch, or directly the 'string', or 'title'), it should be given to it (so added in the controllerParams in activityView).